PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/BubbledGraphTest.java

```diff
@@ -0,0 +1,21 @@
+package nl.tudelft.pl2016gr2.model;
+
+public class BubbledGraphTest extends GraphTest {
+  
+  /**
+   * Setup, creates a new graph.
+   */
```

Comment:
@cbilstra @FarisElghlan I believe you do need to insert `{@inheritDoc}`, as found here http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html